### DISCUSSION TITLE
cache miss, executing ad4ac80904379157   x internal errors encountered: unable to find package manager binary:   | cannot find binary path

### DISCUSSION THREAD

**Discussion by ypavankumar-git** [vercel/turborepo#9419] [Help]
### Summary

 cache miss, executing ad4ac80904379157
  x internal errors encountered: unable to find package manager binary:
  | cannot find binary path

Getting the above error when running the app locally

### Additional information

```js
1. Ran the setup.sh
2. Then ran "npm run dev" and im getting this error
```

**Comment by anthonyshew** [comment]
Can you provide a reproduction?
